bin/webpack-dev-server.js

```diff
@@ -39,6 +39,8 @@ function colorError(useColor, msg) {
 	return msg;
 }
 
+const defaultTo = (v, def) => v == null ? def : v;
```

Comment:
no sense in obfuscating parameters here and one-letter variables make me stabby. please do change `v` to `value`.